addons/auth_totp_mail/models/res_users.py

```diff
@@ -7,6 +7,35 @@
 class Users(models.Model):
     _inherit = 'res.users'
 
+    def write(self, vals):
+        res = super().write(vals)
+
+        if 'totp_secret' in vals:
+            if vals.get('totp_secret'):
+                self._notify_security_setting_update(
+                    _('Security Update: 2FA Activated'),
+                    _("This email to let you know that two-factor authentication has been activated on your account."),
```

Comment:
shouldn't this text be in the qweb template instead?

Reply:
Hello @mart-e 

As this part is customized for each different notification "type", I preferred to keep it out of the templates.

I mean we could have a bunch of t-if/t-else in the templates and some xpathing in sub-modules, but it seems more tedious.